ntoskrnl/kdbg/kdb_cli.c

```diff
@@ -1989,9 +2120,9 @@ KdbpCmdMod(
     {
         if (!KdbpSymFindModule(NULL, 0, &LdrEntry))
         {
-            ULONG_PTR ntoskrnlBase = ((ULONG_PTR)KdbpCmdMod) & 0xfff00000;
+            ULONG_PTR ntoskrnlBase = (ULONG_PTR)__ImageBase;
             KdbpPrint("  Base      Size      Name\n");
-            KdbpPrint("  %08x  %08x  %s\n", ntoskrnlBase, 0, "ntoskrnl.exe");
+            KdbpPrint("  %p  %p  %s\n", (PVOID)ntoskrnlBase, NULL, "ntoskrnl.exe");
```

Comment:
Here, for the size (second displayed field) you show it "as a pointer", but below (line 2135) you keep the %08x format?

Reply:
Fixed